cudax/test/binutils/demangle.pass.cpp

```diff
@@ -0,0 +1,65 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of CUDA Experimental in CUDA C++ Core Libraries,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+// This test uses assert(...) for checking results
+#undef NDEBUG
```

Comment:
I'd prefer to see something more robust than relying on `#undef NDEBUG`/`assert`. Even just `exit(1)` would be better IMO.